### DISCUSSION TITLE
Impossibile collegare dispositivi su Mint 21 Mate

### DISCUSSION THREAD

**Discussion by IvanDeFelice76** [blueman-project/blueman#2281] [Q&A]
![Schermata a 2024-02-26 15-04-52](https://github.com/blueman-project/blueman/assets/26720956/1da87996-b312-4c08-ac06-5d09d5a09438)

Impossibile collegare dispositivi su Mint 21 Mate

**Comment by cschramm** [comment]
`org.bluez.Error.NotReady` means that the adapter is not powered
